File: python/problem001.py
def solution(n):
    list_num = [num for num in range(n) if (num % 3 == 0) or (num % 5 == 0)]
    return sum(list_num)

n = int(input("Escribe el limite de los numeros a evaluar:"))
sum = solution(n)
print('La sumatoria de los multiplos de 3 y 5 por debajo de ' + str(n) + ' es: ' + str(sum))

File: C#/src/ProjectEulerSolutions/Problem001.cs
namespace ProjectEulerSolutions.EulerProblems
{
    public class Problem001
    {
        public static int Solution(int len)
        {
            int sum = 0;
            for (int i = 0; i < len; i++)
            {
                if (i % 3 == 0 || i % 5 == 0)
                    sum += i;
            }
            return sum;
        }
    }
}
